voltixApp/VoltixApp/Views/Vault/VaultAssetsView.swift

```diff
@@ -27,10 +26,6 @@ struct VaultAssetsView: View {
                 .onAppear {
                     loadData()
                 }
-            }
-            .refreshable {
-                loadData()
```

Comment:
why we remove this? I guess it doesn't matter too much anyway , cause @Rockindash is changing it